apps/TestHead.cmake

```diff
@@ -5,7 +5,11 @@ set(OpenMEEG_BINARY_DIR ${CMAKE_BINARY_DIR}) # XXXX this has to be done differen
 file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/tests/) # XXXX this has to be done differently
 
 
-add_test(CLEAN-TESTS ${CMAKE_COMMAND} -P ${OpenMEEG_SOURCE_DIR}/tests/clean_tests.cmake)
+add_test(
+    NAME CLEAN-TESTS
+    WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/../tests/"
```

Comment:
later we can talk why `/../` is needed. (I mean, is needed 'cos `CMAKE_CURRENT_BINARY_DIR` points to apps, but I'm not sure if this is correct) but this will be done some other day.